api/src/main/java/by/borge/jarl/ResultSet.java

```diff
@@ -38,14 +60,33 @@ public static ResultSet fromResultsList(List<Map<String, ?>> list) {
                 .collect(Collectors.toList()));
     }
 
+    /**
+     * Gets the list of {@link Result}s in this {@link ResultSet}.
+     *
+     * @return the list of {@link Result}s in this ResultSet
+     */
     public List<Result> getResults() {
         return results;
     }
 
+    /**
+     * Gets the first {@link Result} in this {@link ResultSet}, or <code>null</code> if empty.
+     * @return
+     */
     public Result getFirst() {
```

Comment:
Should we use `@Nullable` annotations for this API? I think that'd be nice.

Reply:
Not using it OTOH, makes libraries pretty much unusable in Kotlin... so I'd take the chance. Let's use this one? https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api/1.3.2

Reply:
(again, I don't mean for this PR.. but we can include it later)